Kickstarter-iOS/Features/LoginTout/Controller/LoginToutViewController.swift

```diff
@@ -138,11 +138,14 @@ public final class LoginToutViewController: UIViewController, MFMailComposeViewC
       }
       |> UILabel.lens.text %~ { _ in Strings.Get_notified_when_your_friends_back_and_launch_projects() }
 
-    _ = self.loginButton
-      |> greyButtonStyle
-      |> UIButton.lens.title(for: .normal) %~ { _ in
-        Strings.login_tout_back_intent_traditional_login_button()
-      }
+    if self.viewModel.outputs.loginWithOAuthEnabled {
+      // TODO: Add and translate a new version of this string for this page.
```

Comment:
This will be a separate web PR + a follow-up to this PR once the strings are finished.